src/mgmt_conn.c

```diff
@@ -1278,6 +1278,24 @@ uzfs_zinfo_rebuild_from_clone(zvol_info_t *zinfo)
 	return (uzfs_zinfo_rebuild_start_threads(&mack, zinfo, 1));
 }
 
+/*
+ * This function returns TRUE if success can be sent to target
+ */
+static boolean_t
+can_start_rebuild_return_success(zvol_info_t *zinfo, char *volname)
```

Comment:
can this function be named as is _start_rebuild_on_healthy_replica?

Reply:
I felt `_start_rebuild_on_healthy_replica` this functionality already exists in code(Where we rebuild from its clone). 
How about `can_START_REBUILD_return_success` @vishnuitta suggestion?

Reply:
updated the func name as suggested